### PR TITLE
[CIVIS][DORA] Update doc examples about DORA Metrics incidents API

### PR DESCRIPTION
**Pull request by AdrianLC**
We've done some changes to the endpoint for incidents and the API docs were already updated but there is a manually written example here that was still pending.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/dora_metrics/_index.md [hunk 1]
```diff
@@ -222,12 +222,13 @@ Submit deployment events as described in [deployment frequency](#deployment-freq
 
 You are required to provide the following incident attributes:
 
-- `service`
+- `services` or `team`. At least one of them is required.
```
Comment (kassenq) [inline content/en/dora_metrics/_index.md:225]: "At least one is required."

File: content/en/dora_metrics/_index.md [hunk 2]
```diff
@@ -222,12 +222,13 @@ Submit deployment events as described in [deployment frequency](#deployment-freq
...
 - `started_at`
 
 You can optionally add the following incident attributes:
 
 - `finished_at` for *resolved incidents*. Required for [time to restore service](#time-to-restore-service).
+- `id` provide your own ID. Required to resolve a previously sent incident. When not provided the endpoint will return a random UUID that can be used in the next request.
```
Comment (kassenq) [inline content/en/dora_metrics/_index.md:231]: "`id` for identifying incidents when they are created and resolved. This attribute is user-generated; when not provided, the endpoint returns a Datadog-generated UUID."

re-worded a bit, is this accurate? I don't think we need to say "can be used in the next request"

**Review by kassenq — CHANGES_REQUESTED** [review]

File: content/en/dora_metrics/_index.md
Comment (kassenq) [inline content/en/dora_metrics/_index.md:1]: Let's add details about the Service Validation as well, under the "Set up DORA Metrics" section. Changes below in bold: 

"Services tracked for DORA Metrics **deployment frequency and change lead time** must be registered ... automatically associated with all metrics."

File: content/en/dora_metrics/_index.md [hunk 3]
```diff
@@ -222,12 +222,13 @@ Submit deployment events as described in [deployment frequency](#deployment-freq
...
 You are required to provide the following incident attributes:
 
-- `service`
+- `services` or `team`. At least one is required.
 - `started_at`
 
 You can optionally add the following incident attributes:
 
 - `finished_at` for *resolved incidents*. Required for [time to restore service](#time-to-restore-service).
+- `id` for identifying incidents when they are created and resolved. This attribute is user-generated; when not provided, the endpoint returns a Datadog-generated UUID."
```
Comment (kassenq) [inline content/en/dora_metrics/_index.md:231]: remove the `"` and we are good :)

**Review by kassenq — APPROVED** [review]

File: content/en/dora_metrics/_index.md (hunk 3 above)
Comment (hestonhoffman) [inline content/en/dora_metrics/_index.md:231]:
```suggestion
- `id` for identifying incidents when they are created and resolved. This attribute is user-generated; when not provided, the endpoint returns a Datadog-generated UUID.
```

**Review by hestonhoffman — APPROVED** [review]

**Pull request merged**